janis_core/tool/test_suite_runner.py

```diff
@@ -230,6 +230,20 @@ def _apply_preprocessor(
             if not output_value:
                 return 0
             value = len(output_value.split("|"))
+        elif test_logic.preprocessor == TTestPreprocessor.ListOfFilesExist:
+            if not output_value:
+                return False
+            files = output_value.split("|")
+            count = 0
+            for file in files:
+                count += 1
```

Comment:
looks like the variable `count` has not been used here. can it be removed?

Reply:
It was only used for testing and I've just removed it.